storage/libradb/src/lib.rs

```diff
@@ -74,6 +77,27 @@ lazy_static! {
     static ref OP_COUNTER: OpMetrics = OpMetrics::new_and_registered("storage");
 }
 
+lazy_static! {
+    pub static ref LIBRA_STORAGE_CF_SIZE_BYTES: IntGaugeVec = register_int_gauge_vec!(
+        // metric name
+        "libra_storage_cf_size_bytes",
+        // metric description
+        "Libra storage Column Family size in bytes",
+        // metric labels (dimensions)
+        &["cf_name"]
+    ).unwrap();
+
+    pub static ref LIBRA_STORAGE_COMMITTED_TXNS: IntCounter = register_int_counter!(
+        "libra_storage_committed_txns",
+        "Libra storage committed transactions"
+    ).unwrap();
+
+    pub static ref LIBRA_STORAGE_LATEST_TXN_VERSION: IntGauge = register_int_gauge!(
+        "libra_storage_latest_transaction_version",
```

Comment:
I thought we were going to do libra::storage::latest_transaction_version?

Reply:
@davidiw Nope, we will use libra_<component>_ as prefix for all the counters. https://prometheus.io/docs/practices/naming/